webrtc.html

```diff
@@ -8621,17 +8646,26 @@ <h2 id="sec.cert-mgmt">Certificate Management</h2>
                 <li>If <var>tones</var> contains any <a>unrecognized</a>
                 characters, <a>throw</a> an <code>InvalidCharacterError</code>.
                 </li>
-                <li>Set the object's <code><a data-for=
-                "RTCDTMFSender">toneBuffer</a></code> attribute to
-                <var>tones</var>.</li>
+                <li>Set the <code><a data-for="RTCDTMFSender">toneBuffer</a></code>
+                attribute of <var>dtmf</var> to <var>tones</var>.</li>
+                <li>Set the <a>[[\Duration]]</a> slot of <var>dtmf</var>
+                to the value of the <code><a data-for=
+                "RTCDTMFSender">duration</a></code> parameter.</li>
+                <li>Set the <a>[[\InterToneGap]]</a> slot of <var>dtmf</var>
+                to the value of the <code><a data-for=
+                "RTCDTMFSender">interToneGap</a></code> parameter.</li>
                 <li>If the value of the <code><a data-for=
                 "RTCDTMFSender">duration</a></code> parameter is less than 40,
-                set it to 40. If, on the other hand, the value is greater than
-                6000, set it to 6000.</li>
```

Comment:
This matches existing formatting, but I noticed a discrepancy between numerical formatting here vs. "`2000` ms" later in this same algorithm.